packages/gatsby-image/README.md

```diff
@@ -94,6 +94,34 @@ module.exports = {
 }
 ```
 
+### Dynamically polyfilling object-fit/object-position
```

Comment:
`### Add support for object-fit/object-position to older browsers`
might be a better title